CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,90 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
+class CircleSpawnController: UIViewController, UIGestureRecognizerDelegate {
 
-	// TODO: Assignment 1
-
-	override func loadView() {
-		view = UIView()
-		view.backgroundColor = .white
-	}
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .systemBackground
+    
+        let doubleTap = UITapGestureRecognizer(target: self, action: #selector(handleDoubleTap(sender:)))
+        doubleTap.delegate = self
+        doubleTap.numberOfTapsRequired = 2
+        view.addGestureRecognizer(doubleTap)
+        
+        let tripleTap = UITapGestureRecognizer(target: self, action: #selector(handleTripleTap(sender:)))
+        tripleTap.delegate = self
+        tripleTap.numberOfTapsRequired = 3
+        view.addGestureRecognizer(tripleTap)
+        
+        doubleTap.require(toFail: tripleTap)
+    }
+    
+    @objc func handleDoubleTap(sender: UITapGestureRecognizer) {
+           
+        let size: CGFloat = 100
+        let circleView = UIView(frame: CGRect(origin: .zero, size: CGSize(width: size, height: size)))
+        circleView.center = sender.location(in: self.view)
+        circleView.backgroundColor = UIColor.randomBrightColor()
+        circleView.layer.cornerRadius = size * 0.5
+        circleView.alpha = 0;
+        circleView.tag = 100;
+        view.addSubview(circleView)
+            
+        circleView.transform = CGAffineTransform(scaleX: 1.5, y: 1.5)
+        let animator = UIViewPropertyAnimator(duration: 0.1 , curve: .easeInOut, animations: {
+            circleView.transform = CGAffineTransform(scaleX: 1.0, y: 1.0)
+            circleView.alpha = 1;
+        })
+        animator.startAnimation()
+        
+        let longPress = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPress(sender:)))
+        circleView.addGestureRecognizer(longPress)
+    }
+    
+    @objc func handleLongPress(sender: UILongPressGestureRecognizer) {
+        guard let circleView = sender.view else { return }
+        let circlePress = sender.location(in: view)
+        var dx = CGFloat()
+        var dy = CGFloat()
```

Comment:
Gdyby `dx` oraz `dy` były zmiennymi w ViewControllerze, a nie w tej metodzie, to by zadziałało 😉